packages/chaire-lib-backend/src/config/ServerConfig.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2024, Polytechnique Montreal and contributors
+ *
+ * This file is licensed under the MIT License.
+ * License text available at https://opensource.org/licenses/MIT
+ */
+import config, { TrRoutingConfig } from './server.config';
+import { RoutingOrTransitMode } from 'chaire-lib-common/lib/config/routingModes';
+
+/**
+ * Helper class to manage accesses to server configuration
+ */
+class ServerConfig {
+    constructor() {
+        // Nothing to do
+        // TODO Currently, the `server.config` file sets the default value,
+        // because there is not that many we want to set, but as we add more,
+        // this class should be responsible to set them
+    }
+
+    /**
+     * Get the routing mode configuration
+     * @param mode The mode to get the configuration for
+     * @returns the project configuration, or undefined if the mode is not configured
+     */
+    getRoutingModeConfig = (mode: RoutingOrTransitMode) => {
+        if (config.routing[mode] === undefined) {
+            console.error('Configuration for routing mode %s not found', mode);
+        }
+        return config.routing[mode];
+    };
+
+    /**
+     * Get the routing engine configuration for a given mode
+     * @param mode The mode to get the configuration for
+     * @param engine The engine to use
+     * @returns
+     */
+    getRoutingEngineConfigForMode = (mode: RoutingOrTransitMode, engine: string) => {
```

Comment:
I feel there's something more typed (and later more generic for the engine management), but we can think about this later

Reply:
Added a TODO